app/Traits/Eloquent/Remover.php

```
@@ -132,30 +97,295 @@ public function removeAllTheEntriesMediaFoldersLocal($projectRef): void
+            if (!empty($objects)) {
+                $deleteResult = $s3Client->deleteObjects([
+                    'Bucket' => $bucket,
+                    'Delete' => [
+                        'Objects' => $objects,
+                        'Quiet' => true // Don't return info about successful deletions
+                    ]
+                ]);
+
+                // Log any errors
+                if (!empty($deleteResult['Errors'])) {
+                    Log::error("S3 bulk delete errors for prefix $prefix:", $deleteResult['Errors']);
```

PHPMD (2.15.0)

Avoid using static access to class '\Log' in method 'bulkDeleteByPrefix'. (Clean Code Rules)

(StaticAccess)